cmd/autoupdate/main.go

```diff
@@ -41,6 +42,7 @@ func main() {
 		fmt.Println("Running in debug mode")
 	}
 
+	util.UpdateGitRepo(context.Background(), CDNJS_PATH)
```

Comment:
that's unreleated but since the old bot isn't running we need to update cdnjs/cdnjs here now